src/libstd/sys/windows/process.rs

```diff
@@ -90,6 +90,12 @@ impl Command {
     pub fn cwd(&mut self, dir: &OsStr) {
         self.cwd = Some(dir.to_os_string())
     }
+    pub fn get_program(&self) -> &OsString {
+        &self.program
+    }
+    pub fn args_iter(&self) -> slice::Iter<OsString> {
```

Comment:
Yes, and the public APIs are a little different b/c they're in theory returning `impl Iterator<Item=OsString>` but the actual type being returned is quite different